homeassistant/components/climate/__init__.py

```diff
@@ -68,8 +69,10 @@
     vol.Required(ATTR_AUX_HEAT): cv.boolean,
 })
 SET_TEMPERATURE_SCHEMA = vol.Schema({
-    vol.Optional(ATTR_ENTITY_ID): cv.entity_ids,
     vol.Required(ATTR_TEMPERATURE): vol.Coerce(float),
+    vol.Optional(ATTR_TARGET_TEMP_HIGH): vol.Coerce(float),
```

Comment:
This should be:

```python
vol.Inclusive(ATTR_TARGET_TEMP_HIGH, 'auto_mode'): vol.Coerce(float),
vol.Inclusive(ATTR_TARGET_TEMP_LOW, 'auto_mode'): vol.Coerce(float)
```
This makes it so they are optional, but required if one of them is filled in.